frontend/src/components/Form.js

```diff
@@ -61,29 +91,33 @@ export default function Form() {
     * @param {Event} event
    */
   function handleFormSubmission(event) {
-    // TODO(#34): Validate input values (before allowing to submit)
+    const currentTimeInMilliseconds = new Date().getTime();
+    if (validateFormInputs(role, productArea, duration,
+        timeAvailableUntil.getTime(), currentTimeInMilliseconds)) {
```

Comment:
If this can't fit on a single line, you can do 

```
(validateFormInputs(
    role, productArea, duration, timeAvailableUntil.getTime(), currentTimeInMilliseconds))
```

and if that still can't fit on a single line then 

```
(validateFormInputs(
    role, 
    productArea, 
    duration,
    timeAvailableUntil.getTime(),
    currentTimeInMilliseconds))
```

Reply:
Used the second version and adjusted it to fix any associated linter errors